build/build-image/cross/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir $TMPDIR \
             github.com/jteeuwen/go-bindata/go-bindata
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v2.2.1; \
+RUN export ETCD_VERSION=v3.0.10; \
```

Comment:
@wojtek-t there have been some pretty nasty bug fixes since 3.0.4 which is why I'm tracking the fix:  https://github.com/coreos/etcd/releases/